index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
```

Comment:
```suggestion
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/reset-css@5.0.1/reset.min.css" />
    <link rel="stylesheet" href="./style.css" />
```
여기에 이런 식으로 구글링한 reset.css를 먹이면, 테그에 기본적으로 먹혀있는 스타일들이 리셋돼서 시맨틱 테그를 이용하기 더 편리해져요!